### DISCUSSION TITLE
help, issues on x86 Mac

### DISCUSSION THREAD

**Discussion by Sonic1paRadise** [wcko87/beatoraja-english-guide#42] [beatoraja questions / troubleshooting]
Picked up _JAVA_OPTIONS: -Dsun.java2d.opengl=true -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel
Error: Could not find or load main class bms.player.beatoraja.MainLoader
Caused by: java.lang.NoClassDefFoundError: javafx/application/Application

**Comment by wcko87** [comment]
This error occurs when you are using a version of Java that doesn't contain JavaFX. I recommend getting the Liberica JRE **(full)**, which contains JavaFX.
More information: https://github.com/wcko87/beatoraja-english-guide/wiki/FAQ-and-Troubleshooting#which-version-of-java-should-i-download

Liberica download: https://bell-sw.com/pages/downloads/

**Comment by Sonic1paRadise** [comment]
I did install full jre

**Reply by wcko87** [reply]
The error is definitely due to a missing javafx, so I think you might have multiple java installations in your pc, and the command file is running the wrong one (one without javafx)
